app/src/main/java/com/col/eventradar/network/LocationResultApi.kt

```diff
@@ -0,0 +1,15 @@
+package com.col.eventradar.network
+
+import com.col.eventradar.network.dto.LocationSearchResultDTO
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface LocationResultApi {
```

Comment:
```suggestion
LocationApi
```
or 
```suggestion
LocationSearchApi
```

because the result is just the response that comes from it